tests/integration/test_utils/data_source_config/mssql.py

```diff
@@ -0,0 +1,63 @@
+from typing import Mapping, Union
+
+import pandas as pd
+import pytest
+
+from great_expectations.compatibility.typing_extensions import override
+from great_expectations.datasource.fluent.interfaces import Batch
+from tests.integration.test_utils.data_source_config.base import (
+    BatchTestSetup,
+    DataSourceTestConfig,
+)
+from tests.integration.test_utils.data_source_config.sql import SQLBatchTestSetup
+
+
+class MSSQLDatasourceTestConfig(DataSourceTestConfig):
+    @property
+    @override
+    def label(self) -> str:
+        return "mssql"
+
+    @property
+    @override
+    def pytest_mark(self) -> pytest.MarkDecorator:
+        return pytest.mark.mssql
+
+    @override
+    def create_batch_setup(
+        self,
+        request: pytest.FixtureRequest,
+        data: pd.DataFrame,
+        extra_data: Mapping[str, pd.DataFrame],
+    ) -> BatchTestSetup:
+        return MSSQLBatchTestSetup(
+            data=data,
+            config=self,
+            extra_data=extra_data,
+        )
+
+
+class MSSQLBatchTestSetup(SQLBatchTestSetup[MSSQLDatasourceTestConfig]):
+    @property
+    @override
+    def connection_string(self) -> str:
+        return "mssql+pyodbc://sa:ReallyStrongPwd1234%^&*@localhost:1433/test_ci?driver=ODBC Driver 17 for SQL Server&charset=utf8&autocommit=true"  # noqa: E501 # it's okay
```

Comment:
are we cool just in-lining these? I guess not an issue since they are local creds?

Reply:
Yup, this is totally safe since it's just a local docker image, and I think it's easiest to read if we just inline it.